### ISSUE TITLE
DiscordAPIError: Unknown Message

### ISSUE THREAD

**Issue by Peterr-K** [LakeYS/Discord-Trivia-Bot#210]
Hi

this I see after time in bot's log, the bot is version from the web site.

thanks!

```
Successfully launched shard 0 of 0
File database '../Questions/' loaded. Parsed 201 questions in 37ms.
Shard 0 connected to 1 server.
(node:1733) UnhandledPromiseRejectionWarning: DiscordAPIError: Unknown Message
    at /home/user/TriviaBot/node_modules/discord.js/src/client/rest/RequestHandlers/Sequential.js:85:15
    at /home/user/TriviaBot/node_modules/snekfetch/src/index.js:215:21
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:1733) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:1733) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by LakeYS** [comment]
Interesting. This might be related to auto-delete-msgs, though it's hard to tell without repro or further stack traces. Any other error/warning messages or side effects when this occurs?

**Comment by jannyba** [comment]
Hi

this is the full error what I see. Settings for "auto-delete-msgs" is "true".
And you are very correct, I tested it further and this is happening, when I deleted manually bot's message.

So this looks like uncatched error, when message to be deleted is already deleted. So this is a bug in discord.js?

thanks

**Comment by LakeYS** [comment]
This should be addressed in 414a586, but some additional testing will be done soon.

**Comment by LakeYS** [comment]
This looks to be caught properly now -- will re-open if this is replicated again.
